rocksdb/tests/test_transaction_db.py

```diff
@@ -0,0 +1,29 @@
+import os
+import sys
+import shutil
+import gc
+import unittest
+import rocksdb
+from itertools import takewhile
+import struct
+import tempfile
+from rocksdb.merge_operators import UintAddOperator, StringAppendOperator
+
+from .test_db import TestHelper
+
+class TestTransactionDB(TestHelper):
+    def setUp(self):
+        TestHelper.setUp(self)
+        opts = rocksdb.Options(create_if_missing=True)
+        tdb_opts = rocksdb.TransactionDBOptions()
+        self.db = rocksdb.TransactionDB(os.path.join("/tmp", "test"), opts, tdb_opts)
+
+    def test_options_used_twice(self):
+        if sys.version_info[0] == 3:
+            assertRaisesRegex = self.assertRaisesRegex
+        else:
+            assertRaisesRegex = self.assertRaisesRegexp
```

Comment:
Not really needed, since the library does not support python2 already. And from what I see in the docs, the `regexp` variant was only an alias?

Reply:
Fixed